Task#1 Enums

KroosT opened the pull request:
Two enums created. Each of it stores types of geometric figures. One has internal _edgesCount()_ method.
TestEnum class has static _edgesCountOrThrow()_ method which is external for enums and returns edges count depends on geometric figure.

KroosT committed:
Enums task moved from another branch.

Added enums Figure1, Figure2 and class TestEnum.
Added variable prefixes. Throw exception instead returning -1.

KroosT committed:
Removed 1. Enums and 2. Generics directories.

KroosT committed:
Renamed method edgesCount.

vmatelsky commented on Tasks/src/Enums/Figure1.java:
```diff
@@ -1,13 +1,17 @@
-    private final int edges;
-    
+
+    private final int mEdges;
```
maybe better name?

KroosT committed:
Renamed variable for better understanding.

KroosT committed:
.gitignore changed

vmatelsky commented on Tasks/src/Enums/Figure1.java:
```diff
@@ -0,0 +1,17 @@
+package Enums;
+
+enum Figure1 {
```
@KroosT note: strive to not use numbers in class names like this: `Figure1` and `Figure2`.
you could put these two enums in different packages

KroosT committed:
Created packages for different realizations.

SimpleFigure package contains simple Figure enum.
ComplexFigure package contains Figure enum with edgesCount() method.
